subsys/bluetooth/audio/endpoint.h

```diff
@@ -79,18 +79,27 @@ struct bt_audio_unicast_group {
 	sys_slist_t streams;
 };
 
+struct bt_audio_broadcast_stream_data {
+#if defined(CONFIG_BT_CODEC_MAX_DATA_COUNT)
+	/** Codec Specific Data count */
+	size_t   data_count;
+	/** Codec Specific Data */
+	struct bt_codec_data data[CONFIG_BT_CODEC_MAX_DATA_COUNT];
+#endif /* CONFIG_BT_CODEC_MAX_DATA_COUNT */
+};
```

Comment:
It looks a bit odd I must say. Does it make sense to define structure this way? It might be empty. Is the implementation usable if CONFIG_BT_CODEC_MAX_DATA_COUNT is 0 or not defined?

Reply:
Good catch. There was actually a place missing a guard for this. 
I added the guard, but maybe we need to revisit this definition. 

Do you have any suggestions?

Reply:
Not really. Let's leave it as it is. Sorry for late response